modules/token/lib/token-apr-handler/fetch-all.ts

```diff
@@ -0,0 +1,16 @@
+import { tokens } from "./tokens";
+import { TokenApr } from "./types";
+
+export const fetchAllAprs = async (): Promise<TokenApr[]> => {
+  const res = await Promise.all(
```

Comment:
any of these fetches can fail for various reasons. I'd rather do that sequential (no need to optimize in the cron) and log proper errors to sentry